src/components/_common/Modal/__test__/index.test.tsx

```diff
@@ -14,25 +14,21 @@ describe('Modal component', () => {
     ReactDOM.createPortal = jest.fn((element: any) => element);
   });
 
-  beforeEach(() => {
-    body?.appendChild(modalRoot);
-  });
+  beforeEach(() => body?.appendChild(modalRoot));
 
-  afterEach(() => {
-    body?.removeChild(modalRoot);
-  });
+  afterEach(() => body?.removeChild(modalRoot));
 
   test('should render Modal component by default', () => {
     const component = create(
-      <Modal isOpen={true} onClose={() => undefined}>
+      <Modal isOpen onClose={() => undefined}>
```

Comment:
We can refactor `onClose` handler here. For example, you can create some utility called, for example,  noop,  that's equal to `() => {}`.

Reply:
thank's, good point